src/pybind/mgr/ansible/module.py

```diff
@@ -346,6 +412,35 @@ def remove_osds(self, osd_ids):
 
         return ansible_operation
 
+    def add_host(self, host, role):
```

Comment:
Signature doesn't match `Orchestrator.add_host`
```suggestion
    def add_host(self, host, role=None):
```

Reply:
As we decided in our last orchestrator meeting , the "role" parameter has been removed.